README.md

```diff
@@ -87,28 +83,25 @@ environment = var.environment
 label_order = var.label_order
 
 google_kms_key_ring_enabled               = true
-google_kms_crypto_key_enabled             = true
+google_kms_crypto_key_enabled             = true    # when prevent_destroy required true use this key (long-lived key)
+google_kms_crypto_key_ephemeral_enabled   = false   # when prevent_destroy required false use this key (short-lived key)
 google_kms_crypto_key_iam_binding_enabled = true
 project_id                                = var.gcp_project_id
 location                                  = var.location
 keys                                      = ["dev"]
-prevent_destroy                           = false
 service_accounts                          = ["serviceAccount:service-xxxxxxxxxxxxxxx.gserviceaccount.com"]
 role                                      = "roles/cloudkms.cryptoKeyEncrypterDecrypter"
 }
 ```
```

Comment:
That  code block part should be in r`eadme.yaml` not in readme.md because or readme generating by using github action